sdk/identity/Azure.Identity/README.md

```diff
@@ -1,99 +1,175 @@
 # Azure Identity client library for .NET
+ The Azure Identity library provides Azure Active Directory token authentication support across the Azure SDK. It provides a set of TokenCredential implementations which can be used to construct Azure SDK clients which support AAD token authentication.  
+ 
+ This library is in preview and currently supports:
+  - [Service principal authentication](https://docs.microsoft.com/en-us/cli/azure/create-an-azure-service-principal-azure-cli)
```

Comment:
Is there another link that we could use here that explains what a Service Principal is and how to relate it to the App?
Something more like what we have for MAnaged Identity authentication